templates/plugin/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "esModuleInterop": true,
+        "allowSyntheticDefaultImports": true,
```

Comment:
This is not needed since `esModuleInterop` already set it to true.

Reply:
@fox1t is not better to wait for [fastify-tsconfig](https://github.com/fastify/tsconfig) land and use here?